packages/staking/src/features/overview/staking-info-card/StakePoolInfo.module.scss

```
@@ -14,6 +14,7 @@
   height: size_unit(8);
   padding: size_unit(1);
   max-width: 100%;
+  margin-left: 16px;
```

to make this consistent with the rest of the styles in this block, I'd consider converting this to `size_unit`s

```suggestion
  margin-left: size_unit(2);
```